### PR TITLE
Bug/removes imageview twice

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Fixed minor bug where imageview will get removed twice on return to screen if there is no instruction image

### CODE REVIEW

**Comment by dephillipsmichael** [comment]
@syoung-smallwisdom not sure why i couldn't add you as a reviewer, but this PR is for you

File: BridgeAppSDK/SBAInstructionStepViewController.swift [hunk 1]
```diff
@@ -109,7 +109,7 @@ open class SBABaseInstructionStepViewController: ORKStepViewController {
         if let image = self.image {
             self.imageView.image = image
         }
-        else {
+        else if self.imageView != nil {
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAInstructionStepViewController.swift]: If you want to change this to possibly be nil, also change the declaration to optional. Currently, this is defined as *expected* to exist.

````
    @IBOutlet public weak var imageView: UIImageView?
````

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]

**Comment by dephillipsmichael** [comment]
FTFY @syoung-smallwisdom

**Review by syoung-smallwisdom — APPROVED** [review]
looks good, except you will have merge failures b/c of some changes I made to the instruction step. Sorry about that. Please see if they resolve your issues.

**Comment by dephillipsmichael** [comment]
This PR is actually not relevant anymore since you made the imageView optional.  :)

**Pull request closed without merging**